http_image_provider/lib/http_image_provider.dart

```diff
@@ -119,3 +124,6 @@ class HttpImage extends ImageProvider<HttpImage> {
   String toString() =>
       '${objectRuntimeType(this, 'HttpImage')}("$url", scale: $scale)';
```

Comment:
I guess this should be renamed too
```suggestion
      '${objectRuntimeType(this, 'HttpImageProvider')}("$url", scale: $scale)';
```